ACPs/TEMPLATE.md

```diff
@@ -1,13 +1,11 @@
-```text
-ACP: <PR Number>
-Title: <ACP title>
-Author(s): <a list of the author's name(s) and optionally contact info: FirstName LastName <foo@bar.com>>
-Discussions-To: <GitHub Discussion URL (POPULATED BY MAINTAINER, DO NOT SET)>
-Status: <Proposed, Implementable, Activated, Stale>
-Track: <Standards, Best Practices, Meta, Subnet>
-Replaces (*optional): <ACP number>
-Superseded-By (*optional): <ACP number>
-```
+| ACP                            | PR Number                                                                                                                                           |
+| :----------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
I think the length will vary depending on the ACP, I recommend keeping this line as minimal as possible:

```suggestion
| :-- | :-- |
```